scripts/start_container.sh

```diff
@@ -3,10 +3,16 @@
 PORT=${PORT:-8888}
```

Comment:
it would be nice if someone with better bash-fu could bind to the "next" open port, like how starting jupyter twice will serve on 8888, then 8889, etc.  I'm usually already running a notebook on 8888, and have to track it down before starting a container (but this is a very small problem!)